user/enterprise/operations-manual.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Travis CI Enterprise Operations manual
+layout: en_enterprise
+
+---
+
+The Operations Manual is a guideline which helps you to resolve problems with your Travis CI Enterprise instance. Our plan is to  enhance this document with frequently occuring support topics over the time. If you would like to see a specific problem case covered here as well, please get in touch with us via [enterprise@travis-ci.com](mailto:enterprise@travis-ci.com).
+
+This document has multiple entrypoints. Each entrypoint is a common problem which we've seen ocurring on a regular basis over the time. The section will guide you through it helping you to resolve it. If the problem couldn't be resolved you'll find instructions on how to proceed at the bottom of the document.
```

Comment:
Perhaps sub `entrypoint` for `topic` or `section`? I feel like "entrypoint" has a less approachable connotation

Reply:
Good point! I'm gonna go for `topic`. `section` feels a bit sterile and disconnected.

Reply:
Also perhaps:
> Each entrypoint is a common problem which we've seen ocurring on a regular basis over the time. The section will guide you through it helping you to resolve it
could be:
> Each topic is an frequently asked question or common problem and includes a guide and some ideas toward resolving issues as they arise.